lld/test/MachO/icf-safe-thunks-dwarf.ll

```diff
@@ -1,115 +1,104 @@
+; NOTE: Code has been autogenerated by utils/update_test_body.py
```

Comment:
Any way to add here the example invocation to regenerate this file ?

Reply:
You just need to run the script with the lit tests as the arguments. You also need to make sure to set `PATH` so that it uses your just-built `clang`. This is explained in the docs.

```
PATH=build/bin:$PATH llvm/utils/update_test_body.py lld/test/MachO/icf-safe-thunks{,-dwarf}.ll
```

https://llvm.org/docs/TestingGuide.html#elaborated-tests